packages/calcite-design-tokens/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to `calcite-design-tokens`
+
+This document contains information about contributing to the `calcite-design-tokens` package. Please read the general [CONTRIBUTING](../../CONTRIBUTING.md) document first, which includes initial setup, commit message conventions, and more.
+
+## Project scripts
+
+### Build
+
+Build the tokens to platform asset formats like CSS, SCSS, and JavaScript.
+
+`npm --workspace=packages/calcite-design-tokens run build`
+
+## Understanding Token Files
+
+All token files may be updated by the Figma Token Studio plugin. These can also be updated in the code directly. Any changes to token files must be reviewed by the Calcite Design Team.
+
+| Name                 | Description                                                                                                       |
+| -------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `src/$metadata.json` | This file is used by Figma Token Studio but is not used to determine final token asset output.                    |
+| `src/$themes.json`   | Each theme defined in this file will output a token asset in each format defined in the token transformer config. |
+| `src/core.json`      | The core design tokens upon which the rest of the tokens are built.                                               |
+| `src/semantic.json`  | The semantic design tokens used by theme files and components.                                                    |
+| `src/component/`     | The component design tokens. These are used by themes.                                                            |
+| `src/calcite/`       | The Calcite themes. This currently consists of a light and dark theme.                                            |
+
+### Be sure your code passes our integration and unit tests
+
+```bash
+# Test current code
+npm --workspace=packages/calcite-design-tokens run test
+```
```

Comment:
Can you move this to the `Project scripts` section above? For example:

```md
## Project scripts

### Test

Run the integration and unit tests to make sure your code passes.

`npm --workspace=packages/calcite-design-tokens run test`
```

It doesn't seem to fit under the `Understanding token files` section.